packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -38,7 +38,22 @@ const nextServerlessLoader: loader.Loader = function() {
     /\\/g,
     '/'
   )
-  return `
+
+  if (page.startsWith('/api')) {
+    return `
+      import { getRouteMatcher } from 'next-server/dist/lib/router/utils/route-matcher';
+      import { getRouteRegex } from 'next-server/dist/lib/router/utils/route-regex';
+      import { apiResolver } from 'next-server/dist/server/api-utils'
+    
+      export default (req, res) => {
+        const params = getRouteMatcher(getRouteRegex('${page}'))(req.url)
```

Comment:
This needs to be gated behind is dynamic:
https://github.com/zeit/next.js/blob/a144f6de3c2695e07807bf1cdcd7ca207ad37cc1/packages/next/build/webpack/loaders/next-serverless-loader.ts#L101